src/CoreWCF.NetTcp/src/CoreWCF/Channels/Framing/RawStream.cs

```diff
@@ -11,16 +11,19 @@ namespace CoreWCF.Channels.Framing
 {
     public class RawStream : Stream
     {
+        private readonly FramingConnection _connection;
```

Comment:
This field isn't used after the class is constructed, curious why it was added?

Reply:
To help in debugging. I can see if the FramingConnection has been reset. I'll stick an `#if DEBUG` around it as it's not needed for normal runs.